### ISSUE TITLE
Improve Pictures

### ISSUE THREAD

**Issue by thoschu96** [mapillary/mapillary_issues#555]
Hey 

i could be very helpfull if you could add support to change the brightness of pictures.
Sometimes pictures, that are taken are very Bright or very Dark .. 
A button to adjust that, could fix that after they got uploaded ..

Cheers
Thorsten

**Comment by thoschu96** [comment]
Maybe to add the function in the App would working too
Like the "deleted pictures you dont want to get upload" .. 
Add an "Magical picture enhancer" as u can see in Iphone ..

**Comment by pbb72** [comment]
Yes, brightness and contrast editing possibilities in the web interface would be great!

**Comment by ybkuang** [comment]
mapillary_issues is no longer an active repository.
Let’s continue discussing features in our forum! We will import this issue as topic to the Mapillary Forum in the coming weeks. We’ll notify here once it’s been done. See you there!
